### PR TITLE
Override OkHttp Version in Otel to Fix CVE-2023-3635

### PR DESCRIPTION
**Pull request by shoothzj**

### CODE REVIEW

**Comment by shoothzj** [comment]
> Error:  We have duplicated artifacts attached.
Error:  Failed to execute goal org.apache.maven.plugins:maven-source-plugin:3.3.0:jar-no-fork (attach-sources) on project buildtools: Presumably you have configured maven-source-plugn to execute twice times in your build. You have to configure a classifier for at least on of them. -> [Help 1]

**Review by eolivelli — APPROVED** [review]
+1

File: pom.xml [hunk 1]
```diff
@@ -1118,6 +1133,18 @@
               </execution>
             </executions>
           </plugin>
+          <!-- skip maven source plugin due to
```
[the remark this answers is not in the document]
Comment (shoothzj) [reply pom.xml:1136]: @eolivelli Ok, I would split this patch into three patch
- Override okhttp version
- Skip maven source plugin
- Add test submodule into ignore list

[the remark this answers is not in the document]
Comment (shoothzj) [reply pom.xml:1136]: #4402 #4403

**Comment by shoothzj** [comment]
okhttp is not introduced in branch-4.16, remove label

**Pull request merged**